src/Middleware/SpaServices.Extensions/src/SpaOptions.cs

```diff
@@ -15,6 +14,7 @@ namespace Microsoft.AspNetCore.SpaServices
     public class SpaOptions
     {
         private PathString _defaultPage = "/index.html";
+        private string _defaultPackageManagerName = "npm";
```

Comment:
Field name should be `_packageManagerName` (it's not the *default*, it's the actual value, right?).

Reply:
Yeah I felt weird about that, but I was trying to stay consistent with the other private field (b/c I'm unfamiliar with the code base). Updated it to `_packageManagerName`.